test/e2e/data/infrastructure-provider-byoh/v1beta1/cluster-with-kcp.yaml

```diff
@@ -28,7 +28,7 @@ metadata:
   name: ${CLUSTER_NAME}
 spec:
   bundleLookupBaseRegistry: projects.registry.vmware.com/cluster_api_provider_bringyourownhost
-  bundleLookupTag: v0.1.0_alpha.2
+  bundleLookupTag: ${BUNDLE_LOOKUP_TAG}
```

Comment:
We might have to update docs as well in these places -
- https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/docs/getting_started.md#create-workload-cluster
- https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/docs/local_dev.md#create-docker-hosts

Reply:
Already update [create-docker-hosts.md](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/docs/local_dev.md#create-docker-hosts). As for [create-workload-cluster.md](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/docs/getting_started.md#create-workload-cluster), it not affected by test/e2e/data/infrastructure-provider-byoh/v1beta1/cluster-template.yaml, it affected by https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/releases/download/<our_new_release>/cluster-template.yaml, Let change it after the release.